src/Makefile

```diff
@@ -682,6 +682,18 @@ ifeq ($(pext),yes)
 	endif
 endif
 
+### 3.7.1 Try to include git commit sha for versioning 
+SHA = $(shell git rev-parse --short HEAD 2>/dev/null)
```

Comment:
I think the names `SHA` and `CDATA` or too generic or unclear. Please consider e.g. `GIT(_COMMIT)_SHA/GIT(_COMMIT)_DATE` or `(LAST_)COMMIT_SHA/(LAST_)COMMIT_DATE`.

Reply:
I tried to keep these short to not litter the Make invocation. So I guess I'll name them `GIT_SHA` and `GIT_DATE`

Reply:
Ok, cool. You forgot one replacement on line 688...

Reply:
Oh shit thanks, overlooked that.